NetworkPolicy/k8s-listener.py

```diff
@@ -101,19 +115,19 @@ def make_rules(addr_scheme, policy_def):
     # Assemble the rules for the top-level policy chain. These rules look at
     # the IP addresses (source and dest) and figure out whether this is
     # incoming our outgoing traffic.
-    u32_in_match  = _make_u32_match(addr_scheme, tenant, from_segment,
-                                    tenant, target_segment)
+    u32_in_match = _make_u32_match(addr_scheme, tenant, from_segment,
+                                   tenant, target_segment)
     u32_out_match = _make_u32_match(addr_scheme, tenant, target_segment,
                                     tenant, from_segment)
 
-    in_chain_name  = policy_chain_name[:-1] + "-IN_"
+    in_chain_name = policy_chain_name[:-1] + "-IN_"
     out_chain_name = policy_chain_name[:-1] + "-OUT_"
 
     rules[policy_chain_name] = [
         _make_rule(policy_chain_name, '-m u32 --u32 "%s" -j %s' %
-                                           (u32_in_match, in_chain_name)),
+                   (u32_in_match, in_chain_name)),
```

Comment:
Same here: This forced indentation makes it look like another parameter to the _make_rule() function when instead it is part of the string formatting (belongs to the second parameter).